test/addons/openssl-binding/test.js

```diff
@@ -1,6 +1,10 @@
 'use strict';
 
 const common = require('../../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  process.exit(0);
```

Comment:
Existing uses of `common.skip()` in other tests `return` afterwards rather than calll `process.exit()`.